pyaqueduct/client/types.py

```diff
@@ -1,7 +1,8 @@
 """Dataclasses for experiment and it's components"""
+
 from dataclasses import dataclass, field
 from datetime import datetime
-from typing import List
+from typing import List, cast
```

Comment:
`cast` looks unnecessary?

Reply:
Yes I am going remove it, pylint caught it too.